patches/000-initial-setup.patch

```diff
@@ -954,14 +954,18 @@ index 3278a7ff30..e4484540c1 100644
  			testEverything(t, priv)
  		})
  	}
-@@ -629,6 +667,10 @@ func TestEncryptOAEP(t *testing.T) {
+@@ -629,6 +667,14 @@ func TestEncryptOAEP(t *testing.T) {
  	n := new(big.Int)
  	for i, test := range testEncryptOAEPData {
  		n.SetString(test.modulus, 16)
 +		if boring.Enabled && !boringtest.Supports(t, "RSA1024") && n.BitLen() < 2048 {
 +			t.Logf("skipping encryption tests with BoringCrypto: too short key: %d", n.BitLen())
 +			continue
 +		}
++ 	if boring.Enabled && i == 1 {
```

Comment:
This effectively means that we are skipping all instances of tests, right (i == 0: skipped above, i = 1: skipped here)? Why not skipping this test entirely?

Reply:
Good point, didn't realize there are only 2 cases. I suppose this way is a bit more future proof if any other cases are added, but might be easier to skip altogether.